app/views/trainees/degrees/_non_uk_degree_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= register_form_with model: [@trainee, @degree], local: true do |f| %>
+<%= register_form_with model: [@trainee, @degree.degree], local: true do |f| %>
```

Comment:
`@degree.degree` is a bit confusing. I suggest an alternative:

```erb
<%= register_form_with model: @degree, url: trainee_degrees_path(@trainee), local: true do |f| %>
```

Reply:
This breaks the form as the class name is DegreeForm, so the fields will be degree_form prefixed.
Best to pass in the real model object to the form builder.